presto-main/src/main/java/com/facebook/presto/sql/planner/QueryPlanner.java

```diff
@@ -388,7 +388,7 @@ private PlanBuilder aggregate(PlanBuilder subPlan, QuerySpecification node)
         boolean needPostProjectionCoercion = false;
         for (FunctionCall aggregate : analysis.getAggregates(node)) {
             Expression rewritten = subPlan.rewrite(aggregate);
-            Symbol newSymbol = symbolAllocator.newSymbol(rewritten, analysis.getType(aggregate));
```

Comment:
The other two changes are a result of failing tests. This one wasn't, but it also seems to need this fix.